Doc/using/configure.rst

```diff
@@ -18,13 +20,23 @@ Features required to build CPython:
 
 * Support for threads.
 
-* OpenSSL 1.1.1 or newer for the :mod:`ssl` and :mod:`hashlib` modules.
+* OpenSSL 1.1.1 is the minimum version and OpenSSL 3.0.9 is the recommended
```

Comment:
Should we capture that 3.0.9 is the recommended minimum, currently this sentence could be read as ==3.0.9, to my eye.

A

Reply:
Ok, I changed the phrasing.